<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wjournal</class>
 <widget class="QMainWindow" name="wjournal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>журнал</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="vstavkadata">
        <property name="text">
         <string>вставить данные</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QWidget" name="gr" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text">
              <string>справочник</string>
             </property>
             <property name="indent">
              <number>7</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <layout class="QVBoxLayout" name="verticalLayout_28">
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>журнал автопарк</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <widget class="QPushButton" name="addrowjournal">
                    <property name="text">
                     <string>добаить строку</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="resettingcelljournal">
                    <property name="text">
                     <string>сброс ячейки</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="resettingrowjournal">
                    <property name="text">
                     <string>сброс строки</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="potvergeniejournal">
                    <property name="text">
                     <string>потверждение</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="deletejournal">
                    <property name="text">
                     <string>пометить на удаление</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QTableView" name="journal"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QGroupBox" name="verticalGroupBox">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item alignment="Qt::AlignTop">
            <widget class="QWidget" name="horizontalWidget_2" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_25">
                <property name="cursor">
                 <cursorShape>UpArrowCursor</cursorShape>
                </property>
                <property name="text">
                 <string>выборка</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <layout class="QHBoxLayout" name="horizontalLayout_18">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_35">
                <item>
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>номер</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>время отпраления</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>время отпраки</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>улица</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>номер авто</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_36">
                <item>
                 <widget class="QLineEdit" name="jid">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="time_in">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="time_out">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="routes_id">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="auto_id">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>меню и справка</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>таблицы</string>
     </property>
     <widget class="QMenu" name="menu_4">
      <property name="title">
       <string>спраочник</string>
      </property>
      <addaction name="avto"/>
     </widget>
     <widget class="QMenu" name="menu_5">
      <property name="title">
       <string>журнал</string>
      </property>
      <addaction name="personal"/>
      <addaction name="routes"/>
     </widget>
     <addaction name="menu_4"/>
     <addaction name="menu_5"/>
     <addaction name="printer"/>
    </widget>
    <addaction name="avtor"/>
    <addaction name="menu_3"/>
    <addaction name="separator"/>
    <addaction name="spravka"/>
    <addaction name="info"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="avtor">
   <property name="text">
    <string>аторизация</string>
   </property>
  </action>
  <action name="spravka">
   <property name="text">
    <string>справка</string>
   </property>
  </action>
  <action name="info">
   <property name="text">
    <string>инфо</string>
   </property>
  </action>
  <action name="avto">
   <property name="text">
    <string>авто</string>
   </property>
  </action>
  <action name="personal">
   <property name="text">
    <string>персонал</string>
   </property>
  </action>
  <action name="routes">
   <property name="text">
    <string>маршруты</string>
   </property>
  </action>
  <action name="printer">
   <property name="text">
    <string>печать таблицы</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
